script/review-logs.ts

```diff
@@ -0,0 +1,17 @@
+/* tslint:disable:no-sync-functions */
+
+import * as fs from 'fs'
+import * as path from 'path'
+const { getUserDataPath } = require('./dist-info')
```

Comment:
👍 I can't rewrite that to TS until the other places it's used are updated, so that will have to do as a stop-gap thingy